hole/play.go

```diff
@@ -356,6 +356,9 @@ func play(
 		cmd.Args = []string{"/usr/bin/tcc", "-run", "-"}
 	case "civet":
 		cmd.Args = []string{"/usr/local/dist/civet", "--comptime", "--no-config", "-"}
+		if hole.ID == "day-of-week" {
```

Comment:
We can't ship such a hacky approach, ICU is used for more than just date logic and I don't want a whitelist of "holes that need it". I tried applying your patch locally and couldn't reproduce the failure on rock paper scissors:
```
console.log new Date('2024-08-03').toLocaleString('en-US', weekday: 'long')

for arg of process.argv[2..]
    console.log arg
```
![image](https://github.com/user-attachments/assets/3e5cdd07-b8e1-4087-b1cd-c3dfe865e7a6)

Reply:
> We can't ship such a hacky approach

Roger that, although _such hacky_ is a bit so so.

> I don't want a whitelist of "holes that need it"

Fully understandable, although there isn't even the slightest forecast of that happening.

> I tried applying your patch locally and couldn't reproduce the failure on rock paper scissors

Well, of course not. In my OP, I referred to DoW. Why would we use `Date` on RPSSL?

The reason I brought up RPSSL was [a failed `e2e` test that was caused by `icu74l.dat`](https://github.com/code-golf/code-golf/actions/runs/10227005938/job/28297956399#step:10:147). Oddly enough, in-game there isn't even an issue with RPSSL, and yet that particular test keeps failing.

In DoW, the line `console.log new Date('2024-08-03').toLocaleString('en-US', weekday: 'long')` causes the fatal error in my OP, while it prints `Saturday` on [civet.dev](https://civet.dev/playground?code=Y29uc29sZS5sb2cgbmV3IERhdGUoJzIwMjQtMDgtMDMnKS50b0xvY2FsZVN0cmluZygnZW4tVVMnLCB3ZWVrZGF5OiAnbG9uZycp) as expected.

I'm not as aware as you are regarding ICU and what its pros and cons are, but I'm not sure how I'd stand against just leaving it like that. Well, I do actually. Believe me, I'm not engaging in a sponsorship to a project to make it _worse_, and all that can be done easier and more efficient shall be done easier and more efficient. My general point is that such an error is unacceptable, and I was five seconds too late to hold you off, unfortunately.

I'll draft it, and look for other approaches.

Reply:
> Next, build the image with the PR and execute it again. Finally, run `make e2e` on the built image.

You have to do these steps without commit [0edc8ec](https://github.com/code-golf/code-golf/pull/1255/commits/0edc8ecdbcb6beb49f6cca6ce2eced04ae5925ef) in order to get the full view.